rbd/rbd.go

```diff
@@ -729,9 +729,17 @@ func (image *Image) Seek(offset int64, whence int) (int64, error) {
 }
 
 // int rbd_discard(rbd_image_t image, uint64_t ofs, uint64_t len);
-func (image *Image) Discard(ofs uint64, length uint64) error {
-	return RBDError(C.rbd_discard(image.image, C.uint64_t(ofs),
-		C.uint64_t(length)))
+func (image *Image) Discard(ofs uint64, length uint64) (int, error) {
```

Comment:
Technically this is an API break as the return is changing but the old call was simply wrong. (Side note: stuff like this is why I will resist calling this code 1.x for a while so we have the opportunity to make sensible  cleanups like this one)

Reply:
Yeah, and I just added a commit that make the `RbdFeature*` constants a `unit64` instead of an `int`. There are probably more things where Golang is more picky than C when it comes to integer comparison.